docs/adc_driver.md

```diff
@@ -18,6 +18,20 @@ Then place this include at the top of your code:
 #include "analog.h"
 ```
 
+### ARM Usage
```

Comment:
This should probably go in the ARM configuration section

https://github.com/qmk/qmk_firmware/blob/32208262357f5c2ef89ede37de12b62bc828e80f/docs/adc_driver.md#configuration

Reply:
I shifted it down to its own subsection but left a link to it in the old spot so it's easy to find without blasting the majority of users.

Reply:
If it's a header (eg `###`), it shouldn't need a link, since a table of contents will be generated automatically.  

You can use `qmk doctor` to verify this.

Reply:
Only level 2 (`##`) headers end up in the TOC (at least using `qmk docs`), the addition is a child of Configuration, leaving it out of TOC. I left the link near the top since it's a precursor to using ADC at all with ARM.

Now that I think of it, it seems most consistent as a subsection of "Usage" since the rest of the page has subsections for AVR and ARM. This page is a bit different than its siblings and I'm not really in a place to editorialize so I'm happy to put the section wherever.